src/containers/userInfo.js

```diff
@@ -2,38 +2,45 @@ import React, { Component } from "react";
 import axios from "axios";
 import ResponsiveLayout from "../components/ResponsiveLayout";
 
-let state = {
-  email: ""
-};
+
 export default WrappedComponent =>
   class userInfo extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
```

Comment:
Using this, the container doesn't share the same state with the other pages. That's why it was outside. Are you overcoming this situation somewhere?

Reply:
Hmm.. I look into that